language/move-prover/spec-lang/src/translate.rs

```diff
@@ -899,6 +900,8 @@ impl<'env, 'translator> ModuleTranslator<'env, 'translator> {
             let translated = et.translate_exp(exp, &BOOL_TYPE);
             et.finalize_types();
             let kind = match kind {
+                PA::SpecConditionKind::Assert => unreachable!(),
+                PA::SpecConditionKind::Assume => unreachable!(),
```

Comment:
How are they unreachable?

Reply:
They are not.  Boogie translation of these expressions will be done in a separate PR.

Reply:
I would use the macro `unimplemented!()` instead

`unreachable!()` is intended for a situation where the case/condition was checked elsewhere

Reply:
Done.